src/pages/faq/tips.md

```diff
@@ -11,7 +11,7 @@ contributors:
 
 ## Updating Dependencies
 
-To update a [npm](https://www.npmjs.com/) dependency, run the following, where `<package-name>` is the package to update:
+To update an [npm](https://www.npmjs.com/) dependency, run the following, where `<package-name>` is the package to update:
 
 ```shell
 $ npm install &lt;package-name>@<version|latest&gt; --save
```

Comment:
I know this is pre-existing, but since you're here would you mind fixing this?

$ npm install <package-name>@<version|latest> --save

Reply:
yes, haha I actually opened the issue for this before I did the overhaul but then I wasn't sure what the formatting was. I realise now there is already naked `<  >` tags on the page so I've fixed it.